docs/src/css/custom.css

```
@@ -437,3 +420,69 @@ h1 { margin: 0.5rem; }
 
 .change-color:hover { text-decoration: none; }
 
+.navbar {
+  background: rgba(255, 255, 255, 0.03);
+  backdrop-filter: blur(12px) saturate(140%);
+  -webkit-backdrop-filter: blur(12px) saturate(140%);
+  border-bottom: 1px solid rgba(221, 224, 237, 0.06);
+  box-shadow: none ;
```

Stylelint (17.11.0)

[error] Duplicate property "box-shadow" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)